app/libs/classes/viva_application.py

```diff
@@ -353,3 +417,23 @@ def _new_re_application(self):
         )
 
         return self._helpers.serialize_object(response)
+
+    def _new_completion(self):
+        try:
+            if self._save_completion_attachments():
```

Comment:
This if statement is not necessary, since the _save_completion_attachments will throw an exception that will be catched here.